docs/operations.md

```diff
@@ -5,7 +5,7 @@ This page lists operational aspects of running Loki in alphabetical order:
 ## Authentication
 
 Loki does not have an authentication layer.
-You are expected to run an authenticating reverse proxy in front of your services, such as an Nginx with basic auth or an OAuth2 proxy.
+You are expected to run an authenticating reverse proxy in front of your services, such as an Nginx with basic auth or an OAuth2 proxy, mutual TLS. See [here](promtail-setup.md#custom-clients-options) for more details about supported authentication methods.
```

Comment:
I'm not sure `, mutual TLS.` makes grammatical sense.  Probably best omitted.

And in the docs we try and put new sentences on new lines to help with future diffs.